src/coreclr/src/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1244,6 +1244,13 @@ private void getFieldInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_MET
             //
 
             targetMethod = methodAfterConstraintResolution;
+            if (!_compilation.CompilationModuleGroup.VersionsWithMethodBody(targetMethod))
+            {
+                // This loosely corresponds to the bit around
```

Comment:
What is the specific bit that this corresponds to?

Reply:
I believe that the undo of the constraint resolution takes place at line 1301,

https://github.com/dotnet/runtime/blob/9380bbf37bb0ba19c7e2671359875c93686b400c/src/coreclr/src/vm/zapsig.cpp#L1301

The line I quoted before corresponds to the "same version bubble" check.